src/Agent.Listener/Program.cs

```diff
@@ -57,47 +57,44 @@ public static int Main(string[] args)
                         return Constants.Agent.ReturnCode.TerminatedError;
                 }
 
-                using (var term = context.GetService<ITerminal>())
```

Comment:
It looks like that's the only thing that changed in this block, is that correct?